validator/src/commands/run/args.rs

```diff
@@ -774,25 +774,19 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
             .validator(is_parsable::<u64>)
             .help("Milliseconds to wait in the TPU receiver for packet coalescing."),
     )
-    .arg(
-        Arg::with_name("tpu_use_quic")
-            .long("tpu-use-quic")
-            .takes_value(false)
-            .hidden(hidden_unless_forced())
-            .conflicts_with("tpu_disable_quic")
-            .help("Use QUIC to send transactions."),
-    )
     .arg(
         Arg::with_name("tpu_disable_quic")
             .long("tpu-disable-quic")
             .takes_value(false)
-            .help("Do not use QUIC to send transactions."),
+            .hidden(hidden_unless_forced())
+            .help("DEPRECATED (UDP support will be dropped): Do not use QUIC to send transactions."),
     )
     .arg(
         Arg::with_name("tpu_enable_udp")
             .long("tpu-enable-udp")
             .takes_value(false)
-            .help("Enable UDP for receiving/sending transactions."),
+            .hidden(hidden_unless_forced())
+            .help("DEPRECATED (UDP support will be dropped): Enable UDP for receiving/sending transactions."),
```

Comment:
these should have been moved to the deprecated args list, not deprecated ad hoc here.

https://github.com/anza-xyz/agave/blob/fecc916333d376cbe2b1013c75f36b99bacae6c4/validator/src/cli.rs#L105-L131